redbot/cogs/mod/kickban.py

```diff
@@ -889,9 +889,10 @@ async def unban(
         """Unban a user from this server.
 
         Requires specifying the target user's ID. To find this, you may either:
-         1. Copy it from the mod log case (if one was created), or
-         2. enable developer mode, go to Bans in this server's settings, right-
-        click the user and select 'Copy ID'."""
+        1. Copy it from the mod log case (if one was created), or
+        2. enable developer mode, go to Bans in this server's settings, right-
+          click the user and select 'Copy ID'.
```

Comment:
Pre-emptive comment - there need to be two spaces here to ensure the text is indented to the same level as the list. Here Discord treats four spaces as a double indent which does not look good. Four would fit in better with the indentation for the actual code but then Discord for some strange reason assumes the first three spaces mean a single indent and the fourth space is shown as an actual space:
![image](https://user-images.githubusercontent.com/51716387/234381781-a602a1b3-12b3-447e-9fff-9f47c03374f6.png)